doc/DSR-start.md

```diff
@@ -1,6 +1,6 @@
 # Tutorial to start with DSR configurations
 
-This tutorial will guide you through the process of deploying and extending DSR configurations (DSRc). DSRc are sets of agents (a kind of RoboComp component) that share a distributed graph (aka G) data structure. See [robocomp-dsr](https://github.com/robocomp/robocomp/tree/development/classes/dsr") for a detailed description, the theory behind DSR and the installation instructions. 
+This tutorial will guide you through the process of deploying and extending DSR configurations (DSRc). DSRc are sets of agents (a kind of RoboComp component) that share a distributed graph (aka G) data structure. See [robocomp-dsr](https://github.com/robocomp/dsr-graph) for a detailed description, the theory behind DSR and the installation instructions. 
```

Comment:
@Sirish07 The link for dsr detailed description is https://github.com/robocomp/robocomp/tree/development/classes/dsr

Reply:
Corrected. Thanks.